js/books.js

```diff
@@ -0,0 +1,28 @@
+// class AwesomeBooks {
+//   books = [];
+//   constructor() {
+//     this.books
+//   }
+//   saveBooks() {
+//    const booksList = JSON.stringify(this.books);
+//    localStorage.setItem('books', booksList);
+//   }
+//   addBook(title, author) {
+//     const book = {
+//       title,
+//       author,
+//       id: Date.now(),
+//     };
+//     this.books.unshift(book);
+//     this.saveBooks();
+//    }
+
+//   removeBook(id) {
+//     this.books = this.books.filter((a) => a.id !== id);
+//     this.saveBooks();
+//   }
+
+//   getStoredBooks() {
+//     this.books = JSON.parse(localStorage.getItem('books'));
+//   }
+// }
```

Comment:
- [ ] Please let's avoid leaving comments in our codebase. Being interrupted by redundant comments can create distractions and make our code looks messy.